addons/website_slides/views/website_slides_templates_homepage.xml

```diff
@@ -0,0 +1,429 @@
+<?xml version="1.0" ?>
+<odoo><data>
+
+<!-- Channels home template -->
+<template id='courses_home' name="Odoo Courses Homepage">
+    <t t-call="website.layout">
+        <t t-call="website_slides.courses_header">
+            <t t-set="header_title" t-value="'Reach new heights'"/>
```

Comment:
Still wonder if this will be translated but I put all my trust in your knowledge sir.

Reply:
To check with mat.